gitevents.js

```diff
@@ -84,6 +86,18 @@ var issueHandler = function issueHandler(event) {
           }
 
         });
+=======
+        if (tito) {
+          console.log('tito ahoy');
+          tito(config.plugins[tito], payload);
+        }
+
+        // events(payload).then(function(event) {
+        //   // !! add event-related plugins here, for example tito !!
+        // }).catch(function(error) {
+        //   rollbar.handleError(error);
+        // });
+>>>>>>> formike
```

Comment:
Expected an identifier and instead saw '>>>'.<br>Expected an operator and instead saw '>>>'.<br>Expected an operator and instead saw '>'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'formike' is not defined.